scripts/install_platform_driver_configs.py

```diff
@@ -76,17 +77,30 @@
 adding new configurations.
 """
 
+
 def install_configs(input_directory, keep=False):
     os.chdir(input_directory)
-
-    keystore = KeyStore()
-    agent = Agent(address=get_address(), identity="platform_driver_update_agent",
-                  publickey=keystore.public, secretkey=keystore.secret,
-                  enable_store=False)
+    ks = KeyStore()
+    publickey = ks.public
+    secretkey = ks.secret
+    identity = "platform_driver_update_agent"
+    capabilities = dict(edit_config_store=dict(identity=PLATFORM_DRIVER))
```

Comment:
I think this would be better to have you use the execute_commands function.  This Modifies the auth file directly and we are trying not to have that be done by any other location than the Auth agent itself.  The creation of the agent can be done using build_agent rathern than the Agent interface.

Reply:
I used `build_platform_wrapper` and `execute_command` to make this script use those tools to make changes to the Volttron platform instead of changing the internals directly.